modules/claims_api/app/sidekiq/claims_api/poa_updater.rb

```diff
@@ -13,14 +13,17 @@ def perform(power_of_attorney_id) # rubocop:disable Metrics/MethodLength
 
       ssn = poa_form.auth_headers['va_eauth_pnid']
       file_number = service.people.find_by_ssn(ssn)[:file_nbr] # rubocop:disable Rails/DynamicFindBy
+      poa_code = extract_poa_code(poa_form.form_data)
 
-      response = service.vet_record.update_birls_record(
-        file_number:,
-        ssn:,
-        poa_code: poa_form.form_data['serviceOrganization']['poaCode']
-      )
+      unless poa_code.nil?
```

Comment:
It is not, however we were still getting one here because of the change to the schema, I added this here to protect in cases like that, rare as it could be, since if it fails silently that just concerns me.